docbook/reference/en/en-US/modules/RESTEasy_Spring_Integration.xml

```diff
@@ -425,91 +376,138 @@ server.deploy(deployment);]]></programlisting>
+                        <entry>Comment</entry>
+                    </row>
+                </thead>
+                <tbody>
+                    <row>
+                        <entry>@RestController</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@RequestMapping</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@GetMapping</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@PostMapping</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@PutMapping</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@DeleteMapping</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@PatchMapping</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@RequestParam</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@RequestHeader</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@MatrixVariable</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@PathVariable</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@CookieValue</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@RequestBody</entry>
+                        <entry/>
+                    </row>
+                    <row>
+                        <entry>@ResponseStatus</entry>
+                        <entry>Only supported as a method annotation</entry>
+                    </row>
+                    <row>
+                        <entry>@RequestParam</entry>
+                        <entry/>
+                    </row>
+                </tbody>
+            </tgroup>
+        </table>
+        <para>
+            Furthermore, the use of <code>org.springframework.http.ResponseEntity</code> as a return value is
+            supported as
+            is the use of <code>javax.servlet.http.HttpServletRequest</code> and
+            <code>javax.servlet.http.HttpServletResponse</code>
+            as method parameters.
+        </para>
+    </section>
+    <section id="SpringBootStarter">
+        <title>Spring Boot starter</title>
+        <para>
+            The RESTEasy project does not include its own component for Spring Boot integration, however PayPal has
```

Comment:
I'm thinking that while we're here we should revisit this section of the doc, given the Spring Boot Integration is now at https://github.com/resteasy/resteasy-spring-boot . Can you do that Weinan please?

Reply:
okay!